assembl/tests/views_tests/test_api2.py

```diff
@@ -34,6 +34,84 @@ def local_to_absolute(uri):
     return uri
 
 
+def test_make_two_columns_and_add_two_synthesis_for_the_columns(test_app,discussion, subidea_1, subidea_1_1, idea_message_column_positive_on_subidea_1_1):
+    """An integration test to verify the creation of of an extra column in a multicolumn."""
+    create_new_column_json="""{
+  "@id": "local:IdeaMessageColumn/%d",
+  "@type": "IdeaMessageColumn",
+  "@view": "default",
+  "color": "green",
+  "header": {
+    "@id": "local:LangString/53",
```

Comment:
Just be aware that this could fail if previous tests create langstrings on the test server and don't delete it. You're forcing the langstring to be created to have this db-id. You can omit it and let the API generate the id for you instead. This applies for all id's that you're passing.